packages/logging-bunyan/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@google-cloud/logging-bunyan",
+  "version": "0.0.0",
```

Comment:
I've usually seen 0.0.1 as the first released version of a module. This may be fine though, I don't think semver has a restriction.